apps/api/app/api.py

```
@@ -2012,6 +2034,52 @@ def add_asset_by_id(*, asset_id: UUID, base_name: str, zf: zipfile.ZipFile) -> O
             return None
         return add_asset_to_zip(asset=asset, base_name=base_name, zf=zf)
 
+    def add_clip_asset(*, clip: dict, key: str, clip_dir: str, name: str, zf: zipfile.ZipFile) -> Optional[str]:
+        asset_id = clip.get(key)
+        if not asset_id:
+            return None
+        try:
+            return add_asset_by_id(asset_id=UUID(str(asset_id)), base_name=f"{clip_dir}/{name}", zf=zf)
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)